src/test/fuzz/asmap_direct.cpp

```diff
@@ -34,7 +34,8 @@ void test_one_input(const std::vector<uint8_t>& buffer)
     if (SanityCheckASMap(asmap, buffer.size() - 1 - sep_pos)) {
         // Verify that for valid asmaps, no prefix (except up to 7 zero padding bits) is valid.
         std::vector<bool> asmap_prefix = asmap;
-        while (!asmap_prefix.empty() && asmap_prefix.size() + 7 > asmap.size() && asmap_prefix.back() == false) asmap_prefix.pop_back();
+        while (!asmap_prefix.empty() && asmap_prefix.size() + 7 > asmap.size() && asmap_prefix.back() == false)
+            asmap_prefix.pop_back();
```

Comment:
Either one line or multiple lines with `{}`, but not this ;)

Reply:
Good point! I'll help `clang-format` with this :)